persistence/persistence-jdbc/src/main/java/uk/gov/justice/services/jdbc/persistence/AbstractJdbcRepository.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.services.jdbc.persistence;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+
+public abstract class AbstractJdbcRepository {
+    static final String JNDI_APP_NAME_LOOKUP = "java:app/AppName";
```

Comment:
This may also be in value producer.

I think you can just go @resource with this to get the value directly...